src/query/Query.utils.ts

```diff
@@ -148,15 +148,29 @@ function applyFilters(query: string, filters: Filter[]) {
         case 'is':
           return isFilterSql(filter);
         default:
-          return `${ident(filter.column)} ${filter.operator} ${filterLiteral(
-            filter.value
-          )}`;
+          return otherFiltersSql(filter);
       }
     })
     .join(' and ')}`;
   return query;
 }
 
+const withFunction = (filter: Filter, values: string, parenthesis: boolean) =>
+  filter.func
+    ? `(select * from ${ident(filter.func)}(${values}))`
```

Comment:
This is the place, where sql query would be different if user selected db function